general/administration/hardware-acceleration.md

```diff
@@ -49,6 +49,117 @@ The hardware acceleration is available immediately for media playback. No server
 
 Each hardware acceleration type, as well as each Jellyfin installation type, requires different setup options before it can be used. It is always best to consult the FFMpeg documentation on the acceleration type you choose for the latest information.
 
+##Install & Setup for Debain/Ubuntu Nvidia NVENC
+First off we need to figure out what the most up to date driver is for your NVIDIA Card is.
+So for this we need to go to the below link.
+
+[NVIDIA Driver Update](https://www.nvidia.com/Download/index.aspx)
+
+1. You'll need to know what NVIDIA Device you have before searching.
+    -IE. i have an MSI Geoforce GTX 970 for the example of looking up im goin to show you my 1650, thats in the computer im typing this out on but the commands should all show the NVIDIA Geoforce GTX 970 for examples. The only thing that will show my 1650 is this output below.
+        If you dont know what kind of GPU you have you can type in terminal ```lspci``` it will have an output like this
```

Comment:
I think you can filter for video devices. Also use only a single backtick die online code

Reply:
i dont know what you mean can you please specify more on what you are trying to say?

Reply:
`lspci` has an argument to filter by device types I think or if it hadn't, at least `lshw` had and either of those work and it minimizes the output to only 2 item for most people, which is probably a good idea